virttest/remote.py

```diff
@@ -961,6 +961,43 @@ def copy_files_from(address, client, username, password, port, remote_path,
                                    "values are scp and rss" % client)
 
 
+def run_remote_cmd(cmd, params, remote_runner=None, ignore_status=True):
+    """
+    A function to run a command on remote host.
+
+    :param cmd: the command to be executed
+    :param params: the parameter for executing
+    :param remote_runner: a remote runner object on remote host
```

Comment:
missing 1 parameters. Others looks good for me

Reply:
updated